cdap-proto/src/main/java/co/cask/cdap/proto/metadata/lineage/FieldOperationOutput.java

```diff
@@ -25,8 +26,11 @@
 import javax.annotation.Nullable;
 
 /**
- * Represents output of a field operation.
+ * Represents the output of the field operation. Output of the field operation
```

Comment:
Represents an output of a field operation:

- in case of a {@link co.cask.cdap.api.lineage.field.WriteOperation}, a list of datasets ({@link EndPoint});
- otherwise a list of field names.

Reply:
done.